src/main/kotlin/WinningResult.kt

```diff
@@ -0,0 +1,45 @@
+import GameResult.Companion.getApposite
+import GameResult.DRAW
+import GameResult.LOSE
+import GameResult.WIN
+import participant.Participant
+import state.Blackjack
+import state.Bust
+import state.Stay
+
+class WinningResult {
```

Comment:
Great job extracting the game result logic into `WinningResult`! 👍

However, I noticed there's no test coverage for this logic yet. 
It would be even better to add tests to make sure this logic is well covered.